catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -132,6 +132,8 @@
 
     private String queryFeedbackEmailDestination;
 
+    private int maximumUploadSize = 1_048_576;
```

Comment:
How did we determine this was the 'correct' maximum size? Seems like this size is a little small.